yt_dlp/extractor/playsuisse.py

```diff
@@ -0,0 +1,296 @@
+# coding: utf-8
+from __future__ import unicode_literals
+import json
+
+from .common import InfoExtractor
+from ..utils import std_headers
+
+
+class PlaySuisseIE(InfoExtractor):
+    _MEDIA_URL = 'https://4bbepzm4ef.execute-api.eu-central-1.amazonaws.com/prod/graphql'
+    _VALID_URL = r'https?://(?:www\.)?playsuisse\.ch/watch/(?P<id1>[0-9]+)'
+    _TESTS = [
+        {
+            'url': 'https://www.playsuisse.ch/watch/763211/0',
+            'md5': '0d716b7a16c3e6ab784ef817ee9a20c1',
+            'info_dict': {
+                'id': '763211',
+                'ext': 'mp4',
+                'title': 'Wilder S01E01 - Knochen',
+                'description': 'md5:8ea7a8076ba000cd9e8bc132fd0afdd8'
+            }
+        },
+        {
+            'url': 'https://www.playsuisse.ch/watch/808675/0',
+            'md5': '7aa043e69fea5044db2da8bb58bca239',
+            'info_dict': {
+                'id': '808675',
+                'ext': 'mp4',
+                'title': 'Der Läufer',
+                'description': 'md5:'
+            }
+        },
+        {
+            'url': 'https://www.playsuisse.ch/watch/817913/0',
+            'md5': '50721c46ca0b3a9836eb61ecb0ed7097',
+            'info_dict': {
+                'id': '42',
+                'ext': 'mp4',
+                'title': 'Nr. 47 S01E01 - Die Einweihungsparty',
+                'description': 'md5:'
+            }
+        }
+    ]
+
+    def _get_media_data(self, media_id):
+        locale = std_headers.get('locale', 'de').strip()
+        # TODO find out why the locale has no effect in request
```

Comment:
if it has no effect, dont use it

Reply:
The point is it _should_ have an effect. The platform offers multiplexes multiple audio streams to the same media id. In the browser this is switchable (sets the header), but it would work in ytdl. I debugged into the request call and didn't see the headers stripped anywhere...

Reply:
I meant remove the unused code. You can keep the `TODO` comment if u think this is something that should be looked into (but explain it better)